consumer/dragonboard/qualcomm-robotics-rb1/installation/le.md

```diff
@@ -0,0 +1,96 @@
+
+***
+
+- **Step 1**: Make sure fastboot is set up on host computer
+- **Step 2**: Connect host computer to Qualcomm Robotics RB1 dev kit
+- **Step 3**: Boot RB1 into fastboot mode
+- **Step 4**: Flash Images
+- **Step 5**: Reboot the development kit
+
+***
+
+#### **Step 1**: Make sure fastboot is set up on host computer.
+##### You can install fastboot from source or using your distro package manager.
+
+- **From source:**
+  - Android SDK “Tools only” for Linux can be downloaded <a href="https://developer.android.com/studio/releases/platform-tools.html" target="_blank">here</a>
+  - The Linux “Tools Only” SDK download does not come with fastboot, you will need to use the Android SDK Manager to install platform-tools.
+  - To do this follow the “SDK Readme.txt” instructions included in your SDK “Tools Only” download.
+
+  If you are still having trouble setting up fastboot, <a href="https://youtu.be/W_zlydVBftA" target="_blank">click here</a> for a short tutorial video
+
+- **Or using a distro package manager:**
+  ```
+  # Red Hat flavour (Fedora, CentOS)
+  $ sudo yum install android-tools
+
+  # Debian (Ubuntu)
+  $ sudo apt-get install android-tools
+
+  # Ubuntu 16.04 (Xenial) and beyond
+  $ sudo apt-get install android-tools-fastboot
+  ```
+
+#### **Step 2**: Connect host computer to Qualcomm Robotics RB1 development kit
+
+- Development kit must be powered off (unplugged from power)
+- Make sure microSD card slot on the development kit is empty
+- Please check the Quick start guide to set the correct dip switches on the development kit.
+- Connect USB to microUSB cable from host computer to the development kit.
+
+#### **Step 3**: Boot Qualcomm Robotics RB1 dev kit into fastboot mode
+
+**Please read all bullet points before attempting**
+
+- Press and hold the Vol (-) button on the dev kit, this is the S4 button. Dev kit should still NOT be powered on
+- While holding the Vol (-) button, power on the development kit by plugging it in
+- Once the development kit is plugged into power, release your hold on the Vol (-) button.
+- Wait for about 20 seconds.
+- Board should boot into fastboot mode.
+
+From the connected host machine terminal window, run the following commands:
+
+```shell
+# Check to make sure device is connected and in fastboot mode
+
+$ sudo fastboot devices
+```
+
+Typically it will show as below
+```shell
+de82318	fastboot
+```
+
+**At this point you should be connected to your Qualcomm Robotics RB1 development kit with a USB to TypeC cable. Your development kit should be booted into fastboot mode and ready to be flashed with the appropriate images.**
+
+#### **Step 4**: Flash Images
+
+The recovery files can be found in [Thundercomm SDK Manager](https://www.thundercomm.com/product/qualcomm-robotics-rb1-platform/).
+
+```shell
+cd <path-to-images>
+fastboot flash abl_a abl.elf
+fastboot flash boot_a sda845-boot.img
```

Comment:
Is `sda845` the actual SoC name? You're using it for RB2 as well.

Reply:
That should be fixed

Reply:
Good point, that's been copied from RB5 pages, the same issue is present over there and should be fixed also.